src/script/components/Modals/PrimaryModal/PrimaryModal.tsx

```diff
@@ -439,14 +449,15 @@ export const PrimaryModalComponent: FC = () => {
                     placeholder={inputPlaceholder}
                     required
                     data-uie-name="backup-password"
-                    onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
-                      updatePasswordWithRules(event.target.value)
-                    }
+                    markInvalid={isBackupPasswordInvalid}
+                    onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                      updatePasswordWithRules(event.target.value);
+                      setIsBackupPasswordInvalid(false);
```

Comment:
Is firing this hook with every single digit change necessary?

I'd love to see a condition that it only fires once (while true)